README.md

```diff
@@ -9,6 +9,9 @@ Budget-App is about building a mobile web application where you can manage your
 # Live Demo
   - [budget-app](https://bush-budget.herokuapp.com/)
 
```

Comment:
- [ ] Kindly record a a video presenting the features of the project. Also make sure that :point_down: 
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
- The video contains a demo of the project, highlighting all the project features.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.